packages/cells/src/widget.ts

```diff
@@ -203,48 +203,15 @@ export class Cell<T extends ICellModel = ICellModel> extends Widget {
     this.addClass(CELL_CLASS);
     const model = (this._model = options.model);
 
-    const contentFactory = (this.contentFactory =
-      options.contentFactory || Cell.defaultContentFactory);
-    this.layout = new PanelLayout();
+    this.contentFactory = options.contentFactory || Cell.defaultContentFactory;
+    this.layout = options.layout ?? new PanelLayout();
     // Set up translator for aria labels
     this.translator = options.translator || nullTranslator;
 
-    // Header
-    const header = contentFactory.createCellHeader();
-    header.addClass(CELL_HEADER_CLASS);
-    (this.layout as PanelLayout).addWidget(header);
-
-    // Input
-    const inputWrapper = (this._inputWrapper = new Panel());
-    inputWrapper.addClass(CELL_INPUT_WRAPPER_CLASS);
-    const inputCollapser = new InputCollapser();
-    inputCollapser.addClass(CELL_INPUT_COLLAPSER_CLASS);
-    const input = (this._input = new InputArea({
-      model,
-      contentFactory,
-      updateOnShow: options.updateEditorOnShow,
-      placeholder: options.placeholder
-    }));
-    input.addClass(CELL_INPUT_AREA_CLASS);
-    inputWrapper.addWidget(inputCollapser);
-    inputWrapper.addWidget(input);
-    (this.layout as PanelLayout).addWidget(inputWrapper);
-
-    this._inputPlaceholder = new InputPlaceholder(() => {
-      this.inputHidden = !this.inputHidden;
-    });
-
-    // Footer
-    const footer = this.contentFactory.createCellFooter();
-    footer.addClass(CELL_FOOTER_CLASS);
-    (this.layout as PanelLayout).addWidget(footer);
-
-    this._isPlaceholder = options.placeholder ?? false;
-
-    // Editor settings
-    if (options.editorConfig) {
-      this.editor.setOptions({ ...options.editorConfig });
-    }
+    this._editorConfig = options.editorConfig ?? {};
+    this._placeholder = true;
+    this._inViewport = false;
+    this.placeholder = options.placeholder ?? true;
```

Comment:
Good point - I updated the migration guide. But it may make more sense to actually take `false` as default value as tempering with cell creation in notebook is less likely in extensions.